File: testLib/src/main/java/me/ycdev/android/lib/test/base/RobolectricBase.java
package me.ycdev.android.lib.test.base;

import org.junit.BeforeClass;
import org.robolectric.shadows.ShadowLog;

public class RobolectricBase {
    @BeforeClass
    public static void setupClass() {
        ShadowLog.stream = System.out;
    }
}

File: testLib/src/main/java/me/ycdev/android/lib/test/base/RobolectricBase.kt
package me.ycdev.android.lib.test.base

import org.junit.BeforeClass
import org.robolectric.shadows.ShadowLog

@Suppress("unused")
open class RobolectricBase {
    companion object {
        @BeforeClass @JvmStatic
        fun setupClass() {
            ShadowLog.stream = System.out
        }
    }
}
